services/addons/images/count_metric/mongo_counter.py

```diff
@@ -0,0 +1,72 @@
+import os
+import logging
+from pymongo import MongoClient
+from datetime import datetime, timedelta
+
+message_types = ["BSM", "TIM", "Map", "SPaT", "SRM", "SSM"]

```

Comment:
Can we make this an environmental variable? If not, can we add PSM to this?

Reply:
It would be best for it to be configurable then. CDOT doesn't need a bunch of PSM counts all at zeroes taking up storage space.

Reply:
Eh, I changed my mind. This can be abstracted later when we don't have a demo so close.

Reply:
Adding PSM.

Reply:
Added